packages/cards/lib/card.json

```diff
@@ -459,6 +469,14 @@
           "x64": "%WINDIR/SysWOW64/bit4ipki.dll"
         }
       }
+    },
+    {
+      "id": "DBADF7126B4E3656FA5E8CF2AE8D712DE953BC98",
```

Comment:
@defacto64 Please check it. Is it correct?

Reply:
I have reviewed https://github.com/PeculiarVentures/webcrypto-local/blob/p11-attributes/packages/cards/lib/card.json and it seems correct to me, as regards reception of my issues #259 and #260.